### DISCUSSION TITLE
netapp harvest manager 1.4.1 - template for 9.10.1P7

### DISCUSSION THREAD

**Discussion by loryn78** [NetApp/harvest#1330] [General]
hey, I have an ancient netapp harvest manager - 1.4.1 - Is working fine until ontapp 9.7. Until upgrade will happen, is there a way to find/get a template file which support 9.10.1 and latest ontapp versions.
I did a copy of contents from cdot-9.7.0.conf into cdot-9.10.0. conf,  no luck with this approach.

**Comment by ruanruijuan** [comment]
Harvest 1.6.x and earlier were out of support May 2021. This github project is for Harvest 2.0 (version 21.05 onwards). We recommend you to install Harvest latest version to monitor your ONTAP Systems.
